deps/wazuh_testing/wazuh_testing/qa_docs/schema.yaml

```diff
@@ -173,9 +173,12 @@ predefined_values:
   tags:
     - active_response
     - agentd
+    - alas_feeds
     - alerts
     - analysisd
     - api
+    - archlinux
+    - archlinux_feeds
     - ar_analysisd
     - ar_execd
```

Comment:
```suggestion
    - ar_analysisd
    - ar_execd
    - archlinux
    - archlinux_feeds
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/92b4e92a92d158ca4a5355cb342855605b51e726)